src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/ServiceJourneys/Editor/index.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { selectIntl } from 'i18n';
+import { SuccessButton } from '@entur/button';
+import { Dropdown } from '@entur/dropdown';
+import { Tooltip } from '@entur/tooltip';
+import { InputGroup, TextField } from '@entur/form';
+import PageHeader from 'components/PageHeader';
+import { ServiceJourney, StopPoint } from 'model';
+import BookingArrangementEditor from '../../../../BookingArrangementEditor';
+import PassingTimesEditor from './PassingTimesEditor';
+import DayTypeEditor from './DayTypeEditor';
+import { ORGANISATION_TYPE } from 'model/enums';
+import { isBlank } from 'helpers/forms';
+import messages from '../../messages';
+import { OrganisationState } from 'reducers/organisations';
+
+import './styles.scss';
+
+const DEFAULT_SELECT_LABEL = '--- velg ---';
+const DEFAULT_SELECT_VALUE = '-1';
+
+ServiceJourneyEditor.propTypes = {
+  serviceJourney: PropTypes.instanceOf(ServiceJourney).isRequired,
+  stopPoints: PropTypes.arrayOf(PropTypes.instanceOf(StopPoint)).isRequired,
+  onChange: PropTypes.func.isRequired,
+  onClose: PropTypes.func.isRequired,
+  onSave: PropTypes.func.isRequired,
+  isEditMode: PropTypes.bool
+};
+
+type Props = {
+  serviceJourney: any;
+  stopPoints: any[];
+  onChange: (serviceJourney: any) => void;
+  onClose: () => void;
+  onSave: () => void;
+  isEditMode?: boolean;
+};
+
+export default function ServiceJourneyEditor(props: Props) {
+  const [operatorSelection, setOperatorSelection] = useState(
+    props.serviceJourney.operatorRef
+  );
+  const [validPassingTimes, setValidPassingTimes] = useState(false);
```

Comment:
```suggestion
  const [validPassingTimes, setValidPassingTimes] = useState<boolean>(false);
```

Liker å eksplisitt si typen på disse, siden ellers vil Typescript bare utlede bredest mulig type.